assets/config/bootstrap-variables.scss

```diff
@@ -11,7 +11,7 @@
 // stylelint-disable
 $white:    #fff !default;
 $gray-100: #f8f9fa !default;
-$gray-200: #e9ecef !default;
+$gray-200: #EEEEEE !default;
```

Comment:
Is that a validated color ?

Reply:
*update : I juste update and document the “validated” colors in `dev` (0b4f22cf136029f2c36bfff61aa331b9267fa703)